test/smoke-test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+cd $(mktemp -d)
+
+git clone https://huggingface.co/google/flan-t5-small
+
+/tmp/utils/convert.py --model-path flan-t5-small --model-save-path flan-t5-small-caikit
+
+mv flan-t5-small-caikit /opt/models/
+
+CAIKIT_LOG_FILE=$(mktemp)
+cd /caikit
+./start-serving.sh &>$CAIKIT_LOG_FILE &
+# timeout --signal=SIGINT 30 bash -c "tail -f -n0 $CAIKIT_LOG_FILE | grep 'Caikit Runtime is serving on port'"
+timeout --signal=SIGINT 30 bash -c "grep -m1 'Caikit Runtime is serving on port' <(tail -f $CAIKIT_LOG_FILE)"
+
+grpcurl -plaintext -d '{"text": "At what temperature does liquid Nitrogen boil?"}' -H "mm-model-id: flan-t5-small-caikit" localhost:8085 caikit.runtime.Nlp.NlpService/TextGenerationTaskPredict
```

Comment:
What about adding a check on return code or return value that is not empty?

Reply:
https://github.com/opendatahub-io/caikit-tgis-serving/pull/34/files#diff-ddd7611de37976c7d21b1c8eec98863960dc45a5b6c2ca4b971d06939192be7eR18 is the return code from it

Reply:
Ok now I understand how it works, the script returns the return code of the grpcurl code 👍